test/unit/pixelsDisplayed/index.spec.js

```diff
@@ -0,0 +1,273 @@
+/**
+ * @fileOverview
+ * Unit tests for the Pixels Displayed module.
+ */
+
+var module = require('../../../lib/pixelsDisplayed');
+
+describe('lib/pixelsDisplayed', function () {
+  var pixelsDisplayedVertical;
+  var pixelsDisplayedHorizontal;
+  var sinonSandbox, spy;
+
+  var rect = {};
+
+  var el = {
+    getBoundingClientRect: function () {
+      return rect;
+    }
+  };
+
+  function stubAccessor () {
+    sinonSandbox = sinon.sandbox.create();
+    spy = sinonSandbox.spy();
```

Comment:
What's the intent here; the stub interface includes the spy interface, so anything you want to do with spy can be done with the stub.